app/src/main/java/org/grammatek/simacorrect/spellcheckerservice/SimaCorrectSpellCheckerService.kt

```diff
@@ -77,14 +77,10 @@ class SimaCorrectSpellCheckerService : SpellCheckerService() {
                 val suggestionList: Array<SuggestionsInfo>
                 val suggestionsIndices: Array<YfirlesturAnnotation.AnnotationIndices>
                 try {
-                    var text = textInfos[i].text!!
+                    val text = textInfos[i].text!!
                     if (text.isBlank()) {
                         continue
                     }
-                    if (text.trim()[0].isLowerCase()) {
-                        val index = text.indexOf(text.trim())
-                        text = StringBuilder(text).also { it[index] = text.trim()[0].uppercaseChar() }.toString()
-                    }
```

Comment:
These changes should have been placed in your former merge request instead of here, because this MR seems to imply that it only adds a test suite and doesn't fix any bugs at the same time